mm/ktsan/stack.c

```diff
@@ -3,48 +3,16 @@
 #include <linux/kernel.h>
 #include <linux/stacktrace.h>
 
-/* TODO: remove this function, it cannot be used without frame pointers. */
```

Comment:
I think it makes sense to leave this function for debugging purposes.
Maybe put it under KT_DEBUG.